src/routes/Agreements.js

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { injectIntl, intlShape } from 'react-intl';
-
+import { FormattedDate, FormattedMessage } from 'react-intl';
 import { SearchAndSort } from '@folio/stripes/smart-components';
-import { getSASParams } from '@folio/stripes-erm-components';
-
+import { getSASParams, Tags } from '@folio/stripes-erm-components';
```

Comment:
Make sure to update the version of stripes-erm-components you're relying on for the `Tags` component after you bump the version.